index.js

```diff
@@ -4,6 +4,7 @@ const path = require('path');
 const app = express();
 const cors = require('cors');
 const bodyParser = require('body-parser');
+const favorites = require('./lib/routes/api/v1/favorites')
```

Comment:
This looks so much better, using MVC conventions makes everything look cleaner and is definitely more organized.